util/validate2baseline

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+
+require "optparse"
+require "cli"
+
+DEFAULT_TEST_TYPE = "apples to apples"
+DESCRIPTION = <<-DESC
+  `validate2baseline` validates performance results data against baseline data.
```

Comment:
There should be an update to one of our docs about how this script fits into our workflow, and such...

Reply:
I have no idea how this fits into our workflow.  I am happy to ticket such documentation, but I have no context for what workflows/efforts this script are meant to support.  The ticket for this PR and all of its related tickets are simply flagged as tech-debt.  @RandellP do you have insight into what epic(s) such documentation should be added to?

Reply:
Well it is for use comparing to a baseline after a run is done.  Like say you forgot to set the baseline.  Or you want to compare to an additional.  But at the very list the base usage should be covered in something like report_utils...  https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/docs/report_utils.md  
The long term goal is to use this in CI.  So we only have to characterize a build once, then have separate jobs that compare that build to multiple pipelines.

Reply:
[SLV-701](https://tickets.puppetlabs.com/browse/SLV-701) created for this work.